.idea/inspectionProfiles/Druid.xml

```diff
@@ -70,7 +70,6 @@
       </option>
       <option name="IGNORE_FIELDS_USED_IN_MULTIPLE_METHODS" value="true" />
     </inspection_tool>
-    <inspection_tool class="FieldMayBeFinal" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
Was this change (& the others in this file) intentional?

Reply:
no heh, will fix